proto/xudrpc.proto

```diff
@@ -399,6 +422,23 @@ message Balance {
   uint64 unconfirmed_wallet_balance = 6 [json_name = "unconfirmed_wallet_balance"];
 }
 
+message BalanceAlert {
```

Comment:
This could use some better explanation, particularly in terms of how it differs from `ChannelBalanceAlert`.